File: python-solutions/problem009.py
from time import time

def compute():
    for a in range(1, 500):
        for b in range(1, 500):
            c = 1000 - (a + b)
            if a**2 + b**2 == c**2:
                return a*b*c

if __name__ == "__main__":
    start = time()
    answer = compute()
    end = time() - start

    print(f"\nAnswer for Problem 8: {answer}")
    print( f"Time Taken: {end} seconds\n")

File: golang-solutions/problem009.go
package main

import (
	"fmt"
	"time"
)

func main() {
	start := time.Now()
	var a, b, c, temp int

	for i := 1; i < 500; i++ {
		for j := 1; j < 500; j++ {
			temp = 1000 - (i + j)
			if i*i+j*j == temp*temp {
				a, b, c = i, j, 1000-(a+b)
				break
			}
		}
	}

	fmt.Printf("\nAnswer to Problem 9: %d\n", a*b*c)
	fmt.Printf("Time Taken: %v\n\n", time.Since(start))
}
